src/index.js

```diff
@@ -32,14 +40,15 @@ function setupLocalDataTrack() {
 
   window.addEventListener('mousemove', event => {
     const { pageX: x, pageY: y } = event;
-    mouseCoordinates = { x, y };
+    mouseCoordinates = { x: x / window.innerWidth, y: y / window.innerHeight };
 
     if (mouseDown) {
       const color = colorHash.hex(dataTrack.id);
       drawCircle(canvas, color, x, y);
 
+      // Send the drawing coordinate
       dataTrack.send(JSON.stringify({
-        mouseDown,
+        mouseDown: false,
```

Comment:
`mouseDown` is meant to represent the current state of the mouse, but here it is always `false`.